src/utils/price_of_cart/price_of_cart.js

```diff
@@ -0,0 +1,7 @@
+export const price_of_cart = (cart) => {
+  let price = cart.reduce(
+    (accum, ele) => (accum += Number(ele.price) * Number(ele.qnty)),
```

Comment:
Use proper names for object properties to avoid typos and confusion

Reply:
Changed with semantic name